Automattic/wp-calypso
# Add splash screen to Sensei flow

Related to https://github.com/Automattic/evergreen/issues/170.

## Proposed Changes

Adds a splash screen to the Sensei sign-up flow.

## Testing Instructions

1. Run `yarn start`.
2. Go to `http://calypso.localhost:3000/setup/sensei`.
3. Check that the screen matches the  design - km0MaMdbZMb5ySnGP2RSxg-fi-20024055_54108
4. Test that clicking _Get Started_ moves to the `senseiSetup` step of the flow.
5. Test that clicking _Learn More_ goes to https://wordpress.com/create-a-course/.
6. Test the page at smaller resolutions to ensure it looks good.

## Screenshot
![screencapture-wordpress-setup-sensei-intro-2023-07-06-06_57_07](https://github.com/Automattic/wp-calypso/assets/1190420/eab67589-d789-485d-96ce-e6e4170b1c80)

## Pre-merge Checklist
